sse/src/main/scala/akka/stream/alpakka/sse/scaladsl/EventSource.scala

```diff
@@ -94,11 +94,9 @@ object EventSource {
         }
         send(request).flatMap(Unmarshal(_).to[EventSource]).fallbackTo(noEvents)
       }
-      def recover(eventSource: EventSource) = eventSource.recoverWithRetries(1, { case _ => Source.empty })
-      def delimit(eventSource: EventSource) = eventSource.concat(singleDelimiter)
-      Flow[Option[String]]
-        .mapAsync(1)(getEventSource)
-        .flatMapConcat((recover _).andThen(delimit))
+      def recoverAndDelimit(eventSource: EventSource) =
+        eventSource.recoverWithRetries(0, { case _ => Source.empty }).concat(singleDelimiter)
+      Flow[Option[String]].mapAsync(1)(getEventSource).flatMapConcat(recoverAndDelimit)
```

Comment:
To complete this PR you could just set `attempts` to whatever number > 0 and it will work, since `Source.empty` will never fail the number of retries is irrelevant as long as it will happen at least once.